circle.yml

```diff
@@ -0,0 +1,13 @@
+machine:
+  timezone:
+    America/Los_Angeles
+
+test:
+  override:
+    - echo "No tests are needed"
```

Comment:
+1. We should have karma tests for these controllers, and probably a protractor test to verify the actual page.